src/DataSource/Interpreter/BulkXlsxFileInterpreter.php

```diff
@@ -103,6 +103,9 @@ protected function doInterpretFileAndCallProcessRow(string $path): void
 
             // TO use header names as keys, we set the keys for rowData:
             if( !is_null($headerRow) ) {
+                if (count($headerRow) != count($rowData)) {
+                    $rowData = array_pad($rowData, count($headerRow), null);
+                }
                 $rowData = array_combine($headerRow, $rowData);
```

Comment:
Might want to only select the first X ($headerRow.count()) elements of the $rowData in case $rowData has more elements than the header row.